Branch-SDK/Branch-SDK/Requests/BranchShortUrlSyncRequest.m

```diff
@@ -78,36 +79,44 @@ - (NSString *)processResponse:(BNCServerResponse *)response {
 }
 
 - (NSString *)createLongUrlForUserUrl:(NSString *)userUrl {
-    NSMutableString *longUrl = [[NSMutableString alloc] initWithFormat:@"%@?", userUrl];
-    
-    for (NSString *tag in self.tags) {
-        [longUrl appendFormat:@"tags=%@&", tag];
+    NSMutableString *baseUrl = [[NSMutableString alloc] initWithFormat:@"%@?", userUrl];
+    return [BranchShortUrlSyncRequest createLongUrlWithBaseUrl:baseUrl tags:self.tags alias:self.alias type:self.type matchDuration:self.matchDuration channel:self.channel feature:self.feature stage:self.stage params:self.params];
```

Comment:
it looks like `createLongUrlForUserUrl` is being replaced by `createLongUrlWithBaseUrl`, so that all paths that traverse down this will just hit what you wrote, plus `createLinkFromBranchKey` going down `createLongUrlWithBaseUrl`, right?